lib/src/charts/widgets/bar_chart.dart

```diff
@@ -20,6 +19,12 @@ class BarChart extends StatefulWidget {
   /// Height of the Chart
   final double? chartHeight;
 
+  /// Animates the Charts from zero values to given Data when the
+  /// Chart loads for the first time.
+  ///
+  /// Defaults to true.
+  final bool onLoadAnimate;
```

Comment:
Let's rename this to `animateOnLoad`. And add one more bool `animateOnUpdate`. Both should be optional and default to true